src/praisonai/praisonai/bots/_dlq.py

```
@@ -0,0 +1,277 @@
+        deleted) or ``False`` to keep the entry for later. Exceptions
+        from the handler are caught and treated as ``False`` (entry is
+        kept and ``attempts`` incremented).
+
+        Returns ``(succeeded, failed)`` counts.
+        """
+        entries = self.list(limit=limit if limit is not None else self.max_size)
+        succeeded = failed = 0
+        for entry in entries:
+            try:
+                ok = await handler(entry)
+            except Exception as e:  # pragma: no cover — defensive
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)